sklearn/gaussian_process/gpr.py

```diff
@@ -151,6 +164,8 @@ def fit(self, X, y):
         y : array-like, shape = (n_samples, [n_output_dims])
             Target values
 
+        _rng : the RandomState singleton used by np.random.
```

Comment:
If private, it does not deserve documentation. If documented, it should be in the Attributes section of the class docstring, not in the fit docstring.